### PR TITLE
UI/UX Improvements and Bug Fixes

### PR DESCRIPTION
**Pull request by Michael-Obele**
- Fixed bootstrap CDN link.
- Fixed Nav Bar. Used just bootstrap instead of React Bootstrap.
- Revamped the Nav Bar.
- Breakpoint modifications for the elements.
- UI/UX overhaul.
- This should fix issues 10 to 14
- Other Minor, Major Changes.

### CODE REVIEW

**Comment by unnikrishnan2002** [comment]
The improvements looks awesome🎉. Thank you for your valuable contribution. Let me go through files changed.

File: client/package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.1.3",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.3.1",
```
Comment (unnikrishnan2002) [inline client/package.json:16]: Can you just remove this package. And I can merge

**Review by unnikrishnan2002 — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 1 above)
Comment (Michael-Obele) [reply client/package.json:16]: Okay, I will do that now.

**Comment by Michael-Obele** [comment]
[x] Removed react-bootstrap from project!

**Comment by Michael-Obele** [comment]
- [x] Removed react-bootstrap from project!

**Pull request closed without merging**